### PR TITLE
feat(extra): Add wezterm toml config

### PR DESCRIPTION
**Pull request by Uzaaft**
Since someone else is too lazy

### CODE REVIEW

**Comment by luxus** [comment]
right filename would be wezterm/colors/astrodark.toml

**Comment by Uzaaft** [comment]
> right filename would be wezterm/colors/astrodark.toml

I didn't see that in the docs.

File: extras/wezterm_astrodark.toml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+[colors.tab_bar.new_tab_hover]
+bg_color = "#1a1d23"
+fg_color = "#adb0bb"
+intensity = "Normal"
+italic = false
+strikethrough = false
+underline = "None"
+
+[metadata]
+aliases = []
```
Comment (A-Lamia) [inline extras/wezterm_astrodark.toml:77]: under this could you add 
```
name = "Astrodark"
```
 ty :)

**Comment by luxus** [comment]
custom folders are searched in `.config/wezterm/colors`

File: extras/wezterm_astrodark.toml (hunk 1 above)
Comment (Uzaaft) [reply extras/wezterm_astrodark.toml:79]: Added that and you as the author @A-Lamia since you originally created astrodark 🚀

**Review by A-Lamia — APPROVED** [review]

**Comment by A-Lamia** [comment]
> [wezfurlong.org/wezterm/config/appearance.html#defining-a-color-scheme-in-a-separate-file](https://wezfurlong.org/wezterm/config/appearance.html#defining-a-color-scheme-in-a-separate-file)
> 
> > It is recommended that you place your custom scheme in a directory named $HOME/.config/wezterm/colors if you're on a POSIX system.

Isn't that where the end use is supposed to place the file, is there a benefit so setting up the structure like that in the repo, just asking :)

**Comment by Uzaaft** [comment]
> [wezfurlong.org/wezterm/config/appearance.html#defining-a-color-scheme-in-a-separate-file](https://wezfurlong.org/wezterm/config/appearance.html#defining-a-color-scheme-in-a-separate-file)
> 
> > It is recommended that you place your custom scheme in a directory named $HOME/.config/wezterm/colors if you're on a POSIX system.

Isn't this something the user should do?
I've been looking at the github repos that wezterm points to, and very few of them has the theme in a wezterm/colors/ folder.

**Comment by A-Lamia** [comment]
@Uzaaft might be a good idea to change the structure though to be like extras/wezterm/astrodark.toml, i can do that later though so no stress.

**Comment by luxus** [comment]
https://github.com/abzcoding/wezterm is a good example..

**Comment by Uzaaft** [comment]
> @Uzaaft might be a good idea to change the structure though to be like extras/wezterm/astrodark.toml, i can do that later though so no stress.

I'm on it.

**Comment by A-Lamia** [comment]
> [abzcoding/wezterm](https://github.com/abzcoding/wezterm) is a good example..

If you where to clone directly from the repo to the config file though you are not going to get that directory structure, it's going to be like `.config/wezterm/astrotheme/extras/wezterm/colors/astrodark.toml`

**Comment by Uzaaft** [comment]
@A-Lamia Moved the files, all good now?

**Comment by A-Lamia** [comment]
LTGM!

**Comment by Uzaaft** [comment]
Keeping the lua files in case people still want them

**Review by luxus — APPROVED** [review]

**Pull request merged**